### PR TITLE
[BugFix] ENGINE type in information_schema.tables of external table

### PR DESCRIPTION
**Pull request by zombee0**
… Table

## What type of PR is this：
- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Comment by zombee0** [comment]
run starrocks_fe_unittest

File: fe/fe-core/src/main/java/com/starrocks/catalog/Table.java [hunk 1]
```diff
@@ -453,6 +453,18 @@ public String getEngine() {
             return "MySQL";
         } else if (this instanceof SchemaTable) {
             return "MEMORY";
+        } else if (this instanceof HiveTable) {
+            return "Hive";
```
Comment (stephen-shelby) [inline fe/fe-core/src/main/java/com/starrocks/catalog/Table.java:457]: deltalake

**Review by stephen-shelby — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 0 / 14 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/catalog/Table.java|0|14|00.00%|[456, 457, 458, 459, 460, 461, 462, 463, 464, 465, 466, 467, 468, 469]|

**Comment by zombee0** [comment]
run starrocks_be_unittest

**Review by Youngwb — APPROVED** [review]

**Comment by zombee0** [comment]
run starrocks_be_unittest

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Pull request merged**